skills/common/process/batch/runtime/powershell/Scripts/Invoke-BatchOrchestrator.ps1

```
@@ -94,6 +95,76 @@ function Set-BatchStateTransition {
+function Test-GitPrExists {
+    param(
+        [string]$BranchName
+    )
+    $result = & gh pr list --head $BranchName --json number --state all 2>$null
+    if ($LASTEXITCODE -ne 0) { return $null }
+    try {
+        $prs = $result | ConvertFrom-Json
+        if ($prs.Count -gt 0) {
+            return $prs[0].number
+        }
+    } catch { }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)